Tests/WitsmlExplorer.IntegrationTests/Resources/attachment.xml

```diff
@@ -0,0 +1,21 @@
+<attachments xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://www.witsml.org/schemas/1series" xsi:schemaLocation="http://www.witsml.org/schemas/1series  ../xsd_schemas/obj_attachment.xsd" version="1.4.1.1">
```

Comment:
Simplify this to ```<attachments version="1.4.1.1" xmlns="http://www.witsml.org/schemas/1series">``` to keep it similar to the other tests.

Reply:
done